DiceKit/FrequencyDistribution.swift

```diff
@@ -135,7 +135,10 @@ extension FrequencyDistribution {
         }
         
         for (outcome, frequency) in frequenciesPerOutcome {
-            let otherFrequency = x.frequenciesPerOutcome[outcome]!
```

Comment:
Proof that `!` is almost never good.

The logic here was unsound anyway. I was only testing with very similar distributions, where they had the same outcomes.